src/main/org/nlogo/headless/ChecksumsAndPreviews.scala

```diff
@@ -41,6 +41,8 @@ object ChecksumsAndPreviews {
   }
 
   object Previews {
+    def needsManualPreview(previewCommands: String) =
+      previewCommands contains "need-to-manually-make-preview-for-this-model"
```

Comment:
hah. you are right, of course.

yes, I would argue that `containsSlice` is clearer, to Scala eyes — it doesn't lead the careful reader who knows Scala well down the garden path I went down. but then I _would_ say that, to make myself look less foolish for going down that garden path :-)